### PR TITLE
Create popup menu

### PR DESCRIPTION
**Pull request by Verpad**
#51

### CODE REVIEW

File: ProjectSettings/EditorBuildSettings.asset [hunk 1]
```diff
@@ -6,9 +6,9 @@ EditorBuildSettings:
   serializedVersion: 2
   m_Scenes:
   - enabled: 1
-    path: Assets/Project Scope/Scenes/Main Menu.unity
+    path: Assets/Scenes/Main Menu.unity
```
Comment (weterbogov) [inline ProjectSettings/EditorBuildSettings.asset]: @Verpad Are you sure about this? As I remember our scenes are located in `Project Scope` folder. Or It is not true?

Comment (Verpad) [reply ProjectSettings/EditorBuildSettings.asset]: Now project structure looks like this, do you want to change it?
image

Comment (weterbogov) [reply ProjectSettings/EditorBuildSettings.asset]: @Verpad It is ok for current folders structure, but it is better to have prefabs in `project scope/Design` folder.

File: Assets/Project Scope/Scripts/PopupMenuAppear.cs [hunk 2]
```diff
@@ -0,0 +1,22 @@
+﻿using UnityEngine;
+using UnityEngine.UI;
+using System.Collections;
```
[the remark this answers is not in the document]
Comment (Verpad) [reply Assets/Project Scope/Scripts/PopupMenuAppear.cs]: I thinks there is no need in `System.Collections` class

**Pull request merged**